Bottom: ed2e561d826cb0500295bc97e12f6d07ff988ba5
Top:    f0e4866a814196281dfc54c14c41aaa89ae2474a
Author: Lan Yixun (dlan) <dennis.yxun@gmail.com>
Date:   2013-07-24 09:35:26 +0800

From "Lan Yixun (dlan)" <dennis.yxun@gmail.com>

autofs-5.0.7 - fix typo forced-shutdown should be force-shutdown

This will confuse people when they find what shows with "./configure --help"
is different from what is actually used.

edit: imk
Description edited by me.
end: imk

Signed-off-by: Lan Yixun (dlan) <dennis.yxun@gmail.com>


---

diff --git a/configure.in b/configure.in
index 559045a..0013a39 100644
--- a/configure.in
+++ b/configure.in
@@ -363,7 +363,7 @@ fi
 #
 # Enable forced shutdown on USR1 signal (unlink umounts all mounts).
 #
-AC_ARG_ENABLE(forced-shutdown,
+AC_ARG_ENABLE(force-shutdown,
 [  --enable-force-shutdown enable USR1 signal to force unlink umount of any
 			  busy mounts during shutdown],,
 	enableval=no)
